### PR TITLE
feature: optimize update interface

### PR DESCRIPTION
**Pull request by HusterWan**
Signed-off-by: HusterWan <zirenwan@gmail.com>

### Ⅰ. Describe what this PR did

1. optimize update interface, add more test case
2. fix some words spelling mistakes

### CODE REVIEW

File: test/cli_images_test.go [hunk 1]
```diff
@@ -9,12 +9,13 @@ import (
...
 	"github.com/alibaba/pouch/test/command"
 	"github.com/alibaba/pouch/test/environment"
+
 	"github.com/go-check/check"
 	"github.com/gotestyourself/gotestyourself/icmd"
 	"github.com/pkg/errors"
 )
 
-// PouchImagesSuite is the test suite fo help CLI.
+// PouchImagesSuite is the test suite for help CLI.
```
Comment (Letty5411) [inline test/cli_images_test.go]: for help -> for images

File: test/cli_ps_test.go [hunk 2]
```diff
@@ -6,11 +6,12 @@ import (
 
 	"github.com/alibaba/pouch/test/command"
 	"github.com/alibaba/pouch/test/environment"
+
 	"github.com/go-check/check"
 	"github.com/gotestyourself/gotestyourself/icmd"
 )
 
-// PouchPsSuite is the test suite fo help CLI.
+// PouchPsSuite is the test suite for help CLI.
```
Comment (Letty5411) [inline test/cli_ps_test.go]: for ps

File: test/cli_pull_test.go [hunk 3]
```diff
@@ -5,11 +5,12 @@ import (
 
 	"github.com/alibaba/pouch/test/command"
 	"github.com/alibaba/pouch/test/environment"
+
 	"github.com/go-check/check"
 	"github.com/gotestyourself/gotestyourself/icmd"
 )
 
-// PouchPullSuite is the test suite fo help CLI.
+// PouchPullSuite is the test suite for help CLI.
```
Comment (Letty5411) [inline test/cli_pull_test.go]: for pull

File: test/cli_rmi_test.go [hunk 4]
```diff
@@ -5,11 +5,12 @@ import (
 
 	"github.com/alibaba/pouch/test/command"
 	"github.com/alibaba/pouch/test/environment"
+
 	"github.com/go-check/check"
 	"github.com/gotestyourself/gotestyourself/icmd"
 )
 
-// PouchRmiSuite is the test suite fo help CLI.
+// PouchRmiSuite is the test suite for help CLI.
```
Comment (Letty5411) [inline test/cli_rmi_test.go]: for rmi

File: test/cli_start_test.go [hunk 5]
```diff
@@ -7,12 +7,13 @@ import (
...
 	"github.com/alibaba/pouch/test/command"
 	"github.com/alibaba/pouch/test/environment"
+
 	"github.com/go-check/check"
 	"github.com/gotestyourself/gotestyourself/icmd"
 	"github.com/kr/pty"
 )
 
-// PouchStartSuite is the test suite fo help CLI.
+// PouchStartSuite is the test suite for help CLI.
```
Comment (Letty5411) [inline test/cli_start_test.go]: for start, the same for others.

File: test/cli_update_test.go [hunk 6]
```diff
@@ -34,9 +34,42 @@ func (suite *PouchUpdateSuite) SetUpSuite(c *check.C) {
...
+		c.Fatalf("unexpected output %s expected %s\n", string(out), "40")
+	}
+
+	inspectInfo := command.PouchRun("inspect", name).Stdout()
+	metaJSON := &types.ContainerJSON{}
+	if err := json.Unmarshal([]byte(inspectInfo), metaJSON); err != nil {
+		c.Errorf("failed to decode inspect output: %v", err)
+	}
+
+	c.Assert(metaJSON.HostConfig.CPUShares, check.Equals, int64(40))
```
Comment (Letty5411) [inline test/cli_update_test.go]: Please add a `pouch rm` for the container created in this test.

**Comment by allencloud** [comment]
LGTM

**Comment by HusterWan** [comment]
@Letty5411 thanks for you review, this PR has already merged....., i will create a new pr to fix

**Pull request merged**